src/burn/snd/asteroids.cpp

```diff
@@ -20,7 +20,7 @@
  *
  *****************************************************************************/
 

```

Comment:
I'm afraid we can't support this as it will lock out all of the other systems which use fbn.  We have to stay backwards compatible.  sorry about that!

Reply:
You mean xbox/xbox360 ?

Again, those changes are in the scope of C++ language, any way to confirm this ?

Reply:
I know for a fact it won't compile on the compiler iq_132 uses, so this can't happen

Reply:
we have to stay compatible with c++03, nullptr is c++11

Reply:
will revert the nullptr stuff